### ISSUE TITLE
Overzealous lemmatisation of -est words

### ISSUE THREAD

**Issue by adam-ra** [explosion/spaCy#1015]
This happens with some title-cased words, seems not to affect lower-case phrases. Also, does not affect “Honest person”, perhaps there is enough evidence in the training data?

```
Chest pain
 1) ROOT    pain                    NOUN  NN    ROOT
 0) amod   ---- ch                  ADJ   JJS   amod
```

```
Honest pain
 1) ROOT    pain                    NOUN  NN    ROOT
 0) amod   ---- hon                 ADJ   JJS   amod
```

spacy==1.8.1, model en_depent_web_md-1.2.1

**Comment by honnibal** [comment]
The problem's mostly in the POS tagger here: it's identifying these as `JJS`, i.e. superlative adjectives.

Experimental solution: could you try making some calls to `nlp.tagger.update()`? I haven't done much of this sort of "post-training" for the tagger, and it'll be an empirical process. But there should be a way to get decent results doing this.

**Comment by adam-ra** [comment]
I didn't know that an already trained model may be updated this way without losing the existing information, sounds good!

I'll experiment with it within 2 days and will post the results here, thanks!

**Comment by adam-ra** [comment]
This could actually bring lots of improvements in applications with a specific domain (as in my case, medical terms). Will definitely try this ASAP!

**Comment by adam-ra** [comment]
I'm getting `Error allocating memory for feature: 15157689907798343681`. Could you help me?

```
training_data = [
    ('Chest pain', 'NN NN'),
    ('chest pain', 'NN NN'),
    ('Head injury', 'NN NN'),
    ('head injury', 'NN NN'),
]

def update_tagger(tagger, example):
    orth_text, label_text = example
    doc = nlp.tokenizer(orth_text)
    #for word in doc:  This won't help
    #    _ = nlp.vocab[word.orth]
    tags = label_text.split()
    assert len(doc) == len(tags), 'Tokenisation does not match tags for {}'.format(orth_text)
    gold = spacy.gold.GoldParse(doc, tags=tags)
    tagger.update(doc, gold)

def train_tagger(tagger, examples):
    for example in examples:
        update_tagger(tagger, example)
    tagger.model.end_training()

train_tagger(nlp.tagger, training_data)
```

**Comment by honnibal** [comment]
I assume you're on the latest version right? Is it erroring out on `end_training()`, or before?

**Comment by notnami** [comment]
@adam-ra's code works for me, but after training the tags do not match the gold versions.

**Comment by adam-ra** [comment]
@honnibal actually this was under Spacy 1.6.0, still had this version on my Jupyter.
The bug is triggered by tagger.update:
```
<ipython-input-35-13a2a5a84900> in update_tagger(tagger, example)
      7     assert len(doc) == len(tags), 'Tokenisation does not match tags for {}'.format(orth_text)
      8     gold = spacy.gold.GoldParse(doc, tags=tags)
----> 9     tagger.update(doc, gold)
     10 
     11 def train_tagger(tagger, examples):

/home/jupyter/jupyter-notebook/venv/lib/python3.5/site-packages/spacy/tagger.pyx in spacy.tagger.Tagger.update (spacy/tagger.cpp:6924)()

/home/jupyter/jupyter-notebook/venv/lib/python3.5/site-packages/thinc/linear/avgtron.pyx in thinc.linear.avgtron.AveragedPerceptron.updateC (thinc/linear/avgtron.cpp:5314)()

/home/jupyter/jupyter-notebook/venv/lib/python3.5/site-packages/thinc/linear/avgtron.pyx in thinc.linear.avgtron.AveragedPerceptron.update_weight (thinc/linear/avgtron.cpp:5510)()

/home/jupyter/jupyter-notebook/venv/lib/python3.5/site-packages/thinc/linear/avgtron.pyx in thinc.linear.avgtron.AveragedPerceptron.resume_training (thinc/linear/avgtron.cpp:4177)()

MemoryError: Error allocating memory for feature: 15157689907798343681
```

On Spacy 1.8.1 the spacy.en.English().tagger object is None :cry:

**Comment by adam-ra** [comment]
Ok, sorry, the tagger object is not None when initialised the new way: `spacy.load('en_depent_web_md')`
BTW I guess it would be better if an exception was raised rather than a defunct language object was returned.

Anyway, after multiplying the training data it does work!

```
def train_tagger(tagger, examples, reps):
    for rep in range(reps):
        for example in examples:
            update_tagger(tagger, example)
    tagger.model.end_training()

train_tagger(nlp.tagger, training_data, 1000)
```

```
In[14]: [t.tag_ for t in nlp('chest pain')]
Out[14]: ['NN', 'NN']
```

**Comment by ines** [comment]
> Ok, sorry, the tagger object is not None when initialised the new way: `spacy.load('en_depent_web_md')` BTW I guess it would be better if an exception was raised rather than a defunct language object was returned.

Yes, this will definitely be fixed in v2.0!
